test.txt

```diff
@@ -0,0 +1,3 @@
+asdf
+fdas
```

Comment:
Viktor try to save history of comments in his pull-request. And we try it in test-branch.